### PR TITLE
Corral-packages maintenance for rancher rke2/k3s

### PR DESCRIPTION
**Pull request by Josh-Diamond**
- disable locked package variables
- allow publicIP to have valid certs

### CODE REVIEW

File: packages/aws/rancher-k3s.yaml [hunk 1]
```diff
@@ -9,14 +9,5 @@ templates:
...
-    - v1.25.13+k3s1
-    - v1.24.17+k3s1
-    - v1.23.16+k3s1
-  rancher_version:
-    - 2.7.7
-    - 2.7.6
-    - 2.7.5
   cert_manager_version:
-    - 1.8.0
+    - 1.15.0
```
Comment (slickwarren) [inline packages/aws/rancher-k3s.yaml:13]: same question as on the other PR, what rancher versions was this tested with? just confirming it works everywhere

File: templates/k3s/overlay/opt/corral/k3s/init-cluster.sh [hunk 2]
```diff
@@ -7,6 +7,7 @@ config="write-kubeconfig-mode: 644
 cluster-init: true
 tls-san:
   - ${CORRAL_api_host}
+  - ${CORRAL_kube_api_host}
```
Comment (slickwarren) [inline templates/k3s/overlay/opt/corral/k3s/init-cluster.sh:10]: would like to see a run with the change

**Review by slickwarren — COMMENTED** [review]
overall lgtm, just waiting for some runs for final approval

**Review by markusewalker — COMMENTED** [review]
LGTM, just waiting for a job to verify all looks fine from Jenkins output.

**Comment by Josh-Diamond** [comment]
changes drafted here will be included in:
https://github.com/rancherlabs/corral-packages/pull/55

**Pull request closed without merging**